app.js

```diff
@@ -186,3 +190,21 @@ function syncToTarget (event, target) {
     server.replicateNetwork(socket, 'pull')
   }
 }
+
+function getGlobalDatasetCentroid (done) {
+  var bbox = [[-90,90],[-180,180]]
+
+  var stream = osm.queryStream(bbox)
+  var ix = setInterval(function () {
```

Comment:
Why setInterval here? if the first `n` records are not nodes this will take `n/2 seconds` to complete.